db/seeds.rb

```diff
@@ -5,3 +5,51 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+## schema
+  # create_table "categories"
+    # t.string   "name"
+    # t.text     "description"
+
+Category.create!([
```

Comment:
So, there is already a `rake db:populate` task that populates fake category and resource data.  Also, production is already live so not sure the usefulness of a seed file.  Everything else sounds great though of course I haven't seen it yet.  :)